bt_ddos_shield/event_manager.py

```diff
@@ -0,0 +1,36 @@
+from abc import ABC, abstractmethod
+
+class Event:
+    """
+    Class describing event, which happened in the shield.
+    """
+
+    event_description: str
+    exception: Exception
+
+    def __init__(self, event_description: str, exception: Exception = None):
+        """
+        Args:
+            event_description: Description of the event.
+            exception: Exception which caused the event.
+        """
+        self.event_description = event_description
+        self.exception = exception
+
+class EventManager(ABC):
+    """
+    Abstract base class for manager handling events generated by shield.
```

Comment:
I think it should be an EventProcessor, but I'm not entirely sure what this class is for yet

Reply:
I agree, it's better name